pre_commit/constants.py

```diff
@@ -14,13 +14,13 @@
 STAGES = (
     'commit', 'merge-commit', 'prepare-commit-msg', 'commit-msg',
     'post-commit', 'manual', 'post-checkout', 'push', 'post-merge',
-    'post-rewrite',
+    'post-rewrite', 'rebase',
```

Comment:
the `commit` and `push` stages are a mistake that I'm trying to fix -- can you make this match the hook name `pre-rebase` instead of `rebase`?

Reply:
Applied!